solutions/standard/variables.tf

```diff
@@ -215,13 +215,13 @@ variable "kms_endpoint_type" {
 
 variable "kms_key_ring_name" {
   type        = string
-  default     = "sm-cos-key-ring"
+  default     = "secrets-manager-cos-key-ring"
   description = "The name for the new key ring to store the key. Applies only if `existing_secrets_manager_kms_key_crn` is not specified. If a prefix input variable is passed, it is added to the value in the `<prefix>-value` format. ."
 }
 
 variable "kms_key_name" {
   type        = string
-  default     = "sm-cos-key"
+  default     = "secrets-manager-cos-key"
```

Comment:
cos should not be in the name at all. This key used used for encrypting the secrets manager data. There is no COS in this DA. Suggest to rename `secrets-manager-ring`